packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/scale/scale.routing.js

```diff
@@ -0,0 +1,31 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'pci.projects.project.kubernetes.details.nodepools.scale',
+    {
+      url: '/scale?nodePoolId',
+      views: {
+        modal: {
+          component: 'pciProjectKubernetesNodePoolsScaleComponent',
+        },
+      },
+      layout: 'modal',
+      translations: {
+        value: ['.'],
+        format: 'json',
+      },
```

Comment:
You're already loading translations in the module
```suggestion
```

Reply:
Already fixed.